--- spread-sheet-widget-0.3/Makefile.am~	2018-07-04 07:24:48.000000000 +0200
+++ spread-sheet-widget-0.3/Makefile.am	2018-11-09 09:34:18.669724964 +0100
@@ -25,6 +25,7 @@
 
 
 lib_LTLIBRARIES = libspread-sheet-widget.la
+libspread_sheet_widget_la_LIBADD = $(GTK3_LIBS) $(GLIB2_LIBS) -lm
 libspread_sheet_widget_la_CFLAGS = $(GTK3_CFLAGS) $(GLIB2_CFLAGS) -I ${top_builddir}/src
 libspread_sheet_widget_la_SOURCES = \
 	src/ssw-html-parser.c \
